config/lambda/config.go

```diff
@@ -61,6 +61,9 @@ func Configure(p *config.Provider) {
 			SelectorFieldName: "SubnetIDSelector",
 		}
 		delete(r.TerraformResource.Schema, "filename")
+		r.LateInitializer = config.LateInitializer{
+			IgnoredFields: []string{"source_code_hash"},
```

Comment:
The only thing I'd validate here is how this may impact zip based lambdas in S3. I am not sure how setting a late initializer on that may impact this. I think an uptest should alleviate any concerns there though.